rvps/src/cache/local_fs/mod.rs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! This Cache stores RV information inside a local file
+
+use std::path::Path;
+
+use anyhow::*;
+
+use crate::{Cache, ReferenceValue};
+
+/// Local directory path to store the reference values,
+/// which is created by sled engine.
+const FILE_PATH: &str = "./reference_values";
+
+/// `LocalFs` implements [`Cache`] trait. And
+/// it uses rocksdb inside.
+pub struct LocalFs {
+    engine: sled::Db,
+}
+
+impl Default for LocalFs {
+    fn default() -> Self {
+        LocalFs::new(FILE_PATH).expect("Failed to create LocalFs Cache.")
+    }
+}
+
+impl LocalFs {
+    /// Create a new [`LocalFs`] with given
+    /// file storage path.
+    fn new(path: &str) -> Result<Self> {
+        let path = Path::new(path);
+        let engine = sled::open(path)?;
+        Ok(Self { engine })
+    }
+}
+
+impl Cache for LocalFs {
+    fn set(&mut self, name: String, rv: ReferenceValue) -> Result<Option<ReferenceValue>> {
+        let rv_serde = serde_json::to_string(&rv)?;
```

Comment:
I think we can use `serde_json::to_vec()` here and then no need for `rv_serde.to_bytes()` below.

Reply:
Yes, thanks